### PR TITLE
Rock - Paper - Scissors

### PR DESCRIPTION
**Pull request by Joanb2**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,8 +3,95 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Rock, paper, scissors</title>
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
```
Comment (matwjablonski) [inline index.html]: last `/` is unnecessary

File: index.html [hunk 2]
```diff
@@ -3,8 +3,95 @@
...
+	      </span>
+	    </div>
+	    <div class="col-xs-2"></div>
+	    <div class="col-xs-5 text-center">
+	      <span id="js-computerResult">
+	        Computer Score
+	      </span>
+	    </div>
+	  </div>
+	</div>
```
Comment (matwjablonski) [inline index.html]: clean indentation for your html code

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+}
+setGameElements();
+
+var playerPointsElem = document.getElementById('js-playerPoints'),
+    playerNameElem = document.getElementById('js-playerName'),
+    computerPointsElem = document.getElementById('js-computerPoints');
+
+function newGame() {
+  player.name = prompt('Please enter your name', 'Name');
+  if (player.name) {
```
Comment (matwjablonski) [inline js/scripts.js:50]: what will happen when user remove `Name` from prompt and click OK? fix this issue ;)

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    player.score = computer.score = 0;
+    gameState = 'started';
+    setGameElements();
+
+    playerNameElem.innerHTML = player.name;
+  }
+
+}
+
+function playerPick(playerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: unnecessary function

File: js/scripts.js [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    var possiblePicks = ['rock', 'paper', 'scissors'];
+    return possiblePicks[Math.floor(Math.random()*3)];
+}
+
+var playerPickElem = document.getElementById('js-playerPick'),
+    computerPickElem = document.getElementById('js-computerPick'),
+    playerResultElem = document.getElementById('js-playerResult'),
+    computerResultElem = document.getElementById('js-computerResult');
+
+function playerPick(playerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: duplicated above

File: js/scripts.js [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+    playerPickElem.innerHTML = playerPick;
+    computerPickElem.innerHTML = computerPick;
+}
+
+function checkRoundWinner(playerPick, computerPick) {
+  playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+  var winnerIs = 'player';
+
+    if (playerPick == computerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: always compare type of your datas

File: js/scripts.js [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+        player.score++;
+    } else if (winnerIs == 'computer') {
+        computerResultElem.innerHTML = "Win!";
+        computer.score++;
+    }
+    setGamePoints();
+    endGame();
+}
+
+function playerPick(playerPick) {
```
Comment (matwjablonski) [inline js/scripts.js:99]: duplicated third time

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/scripts.js (hunk 7 above)
Comment (Joanb2) [reply js/scripts.js:99]: I can't remove this one. This function is responsible for players scores and picks. If i delete this, this values doesn't appear :)

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**